### ISSUE TITLE
eslint-plugin-tslint should support tslint 6

### ISSUE THREAD

**Issue by crfrolik** [typescript-eslint/typescript-eslint#1808]
This warning is printed by yarn:

```
@typescript-eslint/eslint-plugin-tslint@2.25.0" has incorrect peer dependency "tslint@^5.0.0".
```

tslint 6.1 is now released; `eslint-plugin-tslint` should support it (and should not generate a peer dependency warning)

**Versions**

| package                                   | version |
| ----------------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin-tslint` | `2.25.0` |
| `@typescript-eslint/parser`               | `2.25.0` |
| `TypeScript`                              | `3.8.3` |
| `ESLint`                                  | `6.8.0` |
| `node`                                    | `13.9.0` |
| `npm`                                     | `yarn 1.22.0` |

**Comment by bradzacher** [comment]
Should be a simple PR to bump the version and ensure the tests still work - want to give it a go?

**Comment by crfrolik** [comment]
Sure, I'll do it. Thanks!

**Comment by bradzacher** [comment]
Awesome, thanks!

Small note ahead of time - you will want to ensure it still allows tslint v5 by using the semver `||`
